package.json

```diff
@@ -85,23 +84,23 @@
     "@types/node": "^13.13.1",
     "@types/node-forge": "^0.9.4",
     "@types/webpack": "^4.41.21",
+    "arkit": "^1.6.4",
     "copy-webpack-plugin": "^6.0.2",
     "documentation": "^8.1.2",
     "eslint": "^7.0.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
     "file-loader": "^6.0.0",
-    "grpc-tools": "^1.9.0",
-    "grpc_tools_node_protoc_ts": "^4.1.0",
     "jest": "^25.0.0",
     "npm-dts": "^1.3.0",
     "prettier": "^2.0.5",
-    "protoc": "^1.0.4",
+    "serve": "^11.3.2",
     "threads-plugin": "^1.3.1",
     "ts-jest": "^25.0.0",
     "ts-loader": "^6.2.2",
     "ts-node": "^8.8.2",
     "typedoc": "^0.17.8",
+    "typedoc-plugin-nojekyll": "^1.0.1",
```

Comment:
If we can just use `.nojekyll` file. No need for the plugin. Just add it as part of our docs generation process.

Reply:
ok sure, we can do `touch .nojekyll` which should achieve the same result.